src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/VuapVideos.java

```diff
@@ -0,0 +1,32 @@
+package com.wikia.webdriver.pageobjectsfactory.componentobject.ad;
+
+public class VuapVideos {
+
+  private VuapVideos() {
+    throw new IllegalAccessError("Utility class");
+  }
+
+  public static final String PORVATA_VAST = "porvata_vast";
+
+  public static final String VAST_VIDEO = "', '<?xml version=\"1.0\" encoding=\"UTF-8\"?><VAST xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance" +
+      "\" xsi:noNamespaceSchemaLocation=\"vast_2.0.1-creativeView.xsd\" version=\"2.0\"><Ad id=\"wikia\"><InLine>" +
+      "<AdSystem>Wikia</AdSystem><AdTitle>Wikia Red Test Video</AdTitle><Impression></Impression><Creatives>" +
+      "<Creative><Linear><Duration>00:00:03</Duration><MediaFiles><MediaFile type=\"video/webm\" height=\"360\" width=\"640\" delivery=\"progressive\">" +
+      "<![CDATA[http://public.adeng.wikia-dev.pl/porvata/red.webm]]></MediaFile></MediaFiles></Linear></Creative></Creatives></InLine></Ad></VAST>'";
+
+  public static final String VAST_VIDEO_WITH_TRACKING_EVENTS = "', '<?xml version=\"1.0\" encoding=\"UTF-8\"?><VAST xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance" +
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1213)